### ISSUE TITLE
Add BatchNorm Support from MIOpen

### ISSUE THREAD

**Issue by dagamayank** [ROCmSoftwarePlatform/hipCaffe#23]
New feature request to add MIOpen's batch normalization support.
